release-please-config.json

```diff
@@ -1,18 +1,13 @@
 {
-  "bootstrap-sha": "505795a1312e90f1ce3b59b530622929bc38b4fe",
+  "last-release-sha": "48545ebe503d661af0562b240648253199758d9f",
   "bump-minor-pre-major": true,
-  "packages": {
```

Comment:
Are you sure we can avoid having `packages` in the config? Per the docs, "the minimal content required defines at least one package".

https://github.com/googleapis/release-please/blob/main/docs/manifest-releaser.md#configfile

Reply:
Hmm, good question. I thought it wasn't necessary because all of the same properties are on the root level, but looking at the [JSON Schema](https://github.com/googleapis/release-please/blob/main/schemas/config.json#L405), it is indeed necessary. I'll revert this.